project/sites/pacni/themes/pacni_theme/src/scss/components.scss

```diff
@@ -7,6 +7,7 @@
 
 #header {
   background: resolve('header_bg.jpg') no-repeat top right;
+  margin: 5px auto 0.1rem;
```

Comment:
Should keep the unit consistent and you don't need the 0 in a decimal.
`margin: .5rem auto .1rem`

Reply:
Thanks , this has now been updated